src/js/components/dataDictionary/DataDictionaryPage.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * DataDictionaryPage.jsx
+ * Created by Brett Varney 4/22/2021
+ */
+
+import React from 'react';
+import { PageHeader } from "data-transparency-ui";
+
+import { dataDictionaryPageMetaTags } from 'helpers/metaTagHelper';
+import MetaTags from 'components/sharedComponents/metaTags/MetaTags';
+import Header from "containers/shared/HeaderContainer";
+import Footer from "containers/Footer";
+import { getStickyBreakPointForSidebar } from 'helpers/stickyHeaderHelper';
+
+import DataDictionaryContainer from "containers/dataDictionary/DataDictionaryContainer";
+
+require('pages/dataDictionary/dataDictionaryPage.scss');
+
+export default class DataDictionaryPage extends React.Component {
```

Comment:
Can we just make this a functional component?

Reply:
Why? We have a mix so I did not know there was a standard, and not to argue, but no one has ever explained any functional advantages, and I see several advantages to class-based. Plus that would be another refactor to a working solution.

Reply:
You're right.